qdev_wrappers/customised_instruments/SR86x_ext.py

```diff
@@ -0,0 +1,57 @@
+from qcodes.instrument_drivers.stanford_research.SR86x import SR86x
+
+class SR86x_ext(SR86x):
+    def __init__(self, name: str, address: str, **kwargs: str) ->None:
+        super().__init__(name, address, **kwargs)
+
+       	self.add_parameter(name='iv_gain',
+                           label='I/V Gain',
+                           unit='',
+                           set_cmd=lambda x: x,
+                           get_parser=float)
+
+       	self.add_parameter(name='g',
+                           label='Conductance',
+                           unit='2e^2/h',
+                           get_cmd=self._get_conductance,
+                           get_parser=float)
+
+       	self.add_parameter(name='resistance',
+                           label='Resistance',
+                           unit='Ω',
+                           get_cmd=self._get_resistance,
+                           get_parser=float)
+
+    def _get_conductance(self):
+        V = self.amplitude.get_latest()
+        I = self.X()/self.iv_gain.get_latest()
+        conductance_quantum = 7.7480917310e-5
+        return (I/V)/conductance_quantum
+
+    def _get_resistance(self):
+        V = self.amplitude.get_latest()
+        I = self.X()/self.iv_gain.get_latest()
+        return (V/I)
+
+
+class SR860_ext(SR86x_ext):
+    """
+    The SR860 instrument is almost equal to the SR865, except for the max frequency
+    """
+    def __init__(self, name: str, address: str, reset: bool=False, **kwargs: str) ->None:
+        super().__init__(name, address, max_frequency=5E3, reset=reset, **kwargs)
```

Comment:
Note this is wrong we just merged https://github.com/QCoDeS/Qcodes/pull/1164 to fix that

Reply:
Thanks. My local qcodes version seems to be slightly outdated.